pkg/controller/directvolumemigrationprogress/directpvmigrationprogress_controller_suite_test.go

```diff
@@ -65,11 +66,14 @@ func SetupTestReconcile(inner reconcile.Reconciler) (reconcile.Reconciler, chan
 // StartTestManager adds recFn
 func StartTestManager(mgr manager.Manager, g *gomega.GomegaWithT) (chan struct{}, *sync.WaitGroup) {
 	stop := make(chan struct{})
+	ctx := context.Background()
+	ctx, stopFunc := context.WithCancel(ctx)
 	wg := &sync.WaitGroup{}
 	go func() {
 		wg.Add(1)
-		g.Expect(mgr.Start(stop)).NotTo(gomega.HaveOccurred())
+		g.Expect(mgr.Start(ctx)).NotTo(gomega.HaveOccurred())
 		wg.Done()
 	}()
+	stopFunc() // this is probably the wrong thing
```

Comment:
I think the right thing here is to update this to return the context and have the caller handle contexts. 

Are there too many callers for that?

Reply:
Yeah, to be honest I wasn't really clear on how to fix these tests, hence this comment. The above got the compile errors to disappear, but I know the tests need fixing around this point.